packages/fxa-settings/src/components/App/RememberPassword/index.stories.tsx

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import RememberPassword from '.';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+
+export default {
+  title: 'components/App/RememberPassword',
+  component: RememberPassword,
```

Comment:
What do you think about calling this `ButtonRememberPassword`? Then we can group all our buttons together if they start with that keyword.

Reply:
This should actually be a Link (as you commented), so I've renamed to `LinkRememberPassword`